tests/ssl-opt.sh

```diff
@@ -1896,7 +1896,7 @@ run_test    "Opaque key for client authentication: ECDHE-ECDSA" \
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
 requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
-requires_config_enabled MBEDTLS_ECDSA_C
+requires_config_enabled MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED
```

Comment:
Note: OK, so we were declaring a dependency on `ECDSA_C` for a test that just doesn't require ECDSA at all. Probably copy-paste error. (No change required, just thinking out loud.)